src/get-started/install.md

```diff
@@ -44,9 +44,7 @@ For distributives with `deb`-based package managers (such as Debian or Ubuntu),
 use
 
 ```shell
-add-apt-repository ppa:exonum/rocksdb
-apt-get install build-essential libsodium-dev libsnappy-dev \
-    librocksdb5.17 pkg-config libprotobuf-dev protobuf-compiler
+apt-get install libssl-dev pkg-config
```

Comment:
Don't we need libsodium or librocksdb or snappy :confused: ? 

CC @vitvakatu

Reply:
We do need

Reply:
I don't have any problem with Exonum compilation on ubuntu 16.04 (docker run --rm -it ubuntu:16.04) with only such packages are installed.

Reply:
Probably, except the build time? @aleksuss are they considered as _recommended_ packages (= to speed up the build using pre-built system libraries) or _required_ — but @ivan-ochc has determined it is likely not the case? @Maria-Nosyk Shall we communicate this distinction in the docs or just say — install them all?

Reply:
@dmitry-timofeev @ivan-ochc Actually these packages could build under the hood. But they could be recommended cause could decrease build time of Exonum. I think it would be good to mention this in the manual. Any thoughts ?

Reply:
As far as I can understand, it is desirable to install all the packages. Otherwise the current reduced list of dependencies will hinder the installation process. We can leave the initial list of packages and put an explanation afterwards. It will say that some packages are optional but are required for swift installation process.

Reply:
@Maria-Nosyk exactly 👍🏻